components/Dropdown.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState, useRef } from "react";
+import styles from "./Dropdown.module.css";
+
+const options = [
+  { value: "latest", label: "최신순" },
+  { value: "most_liked", label: "좋아요순" },
+];
+
+interface DropdownProps {
+  value?: { value: string; label: string };
+  onChange?: (option: { value: string; label: string }) => void;
+}
```

Comment:
P2:
해당 코드를 보면 option의 타입이 반복되고 있으니 아래처럼 하시면 더 좋을 것 같습니다.

```suggestion
interface OptionProps { value: string; label: string };

const options: OptionProps[] = [
  { value: "latest", label: "최신순" },
  { value: "most_liked", label: "좋아요순" },
];

interface DropdownProps {
  value?: OptionProps;
  onChange?: (option: OptionProps) => void;
}
```